deploy/infrastructure/providers.tf

```diff
@@ -12,15 +12,27 @@ terraform {
       source  = "hashicorp/aws"
       version = "~> 5.0"
     }
+    {%- if dns_provider == "cloudflare" %}
     cloudflare = {
       source  = "cloudflare/cloudflare"
-      version = "~> 4.22"
+      version = "~> 4.0"
     }
+    {%- endif %}
   }
 }
 
+{%- if dns_provider == "cloudflare" %}
+data "aws_secretsmanager_secret_version" "cloudflare_api_token" {
+  secret_id = "tf_cloudflare_api_token" # follows naming of existing secret in Disco
```

Comment:
Detail: Not sure about the comment mentioning Disco... And the ID don't follow the secret naming convention: https://tamedia.atlassian.net/wiki/spaces/PTS/pages/44304906/Secrets+Naming+convention+and+tags